apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/ServerTestUtils.java

```diff
@@ -954,4 +980,100 @@ public static boolean isValidServerDatabase(String dbUrl) {
     boolean isHsqlDb = dbUrl.contains("hsql");
     return !dbUrl.isBlank() && (isTestContainer || isLocalDb || isHsqlDb);
   }
+
+  /**
+   * Generates a path to use as a filename.
+   *
+   * @param directory the path to where the file should be written
+   * @param endpoint the string to identify which endpoint's response the file contents contain
+   * @return a path to use as a filename
+   */
+  public static Path generateEndpointJsonFileName(Path directory, String endpoint) {
+    return Paths.get(directory.toString(), endpoint + ".json");
+  }
+
+  /**
+   * Reads a file at the specified location.
+   *
+   * @param path the path to the file
+   * @return the contents of the file as a string.
+   */
+  public static String readFile(Path path) {
+    byte[] encoded;
+    try {
+      encoded = Files.readAllBytes(path);
+    } catch (IOException e) {
+      throw new UncheckedIOException("Can't read file at " + path, e);
+    }
+    return new String(encoded, StandardCharsets.UTF_8);
+  }
```

Comment:
What's wrong with Files.readString()?

Reply:
Old code from Java11, probably. Updated it.